Dynavity/Dynavity/view/UmlSideMenuView.swift

```diff
@@ -6,7 +6,6 @@ struct UmlSideMenuView: View {
     private let umlCloseButtonOffset: CGFloat = -12.0
     private let shapePreviewSize: CGFloat = 80.0
 
-    // TODO: Move this to the model, so that view will always be in sync
```

Comment:
I don't think that this should be in the model since this is effectively just a list of UML elements which should appear in the side menu. Either way, to get a list of types which conform to `UmlElementProtocol` automatically, we would probably need to use some form of type reflection (such as https://gist.github.com/codelynx/bb72bf0bed58a327ce1dbe6639d0369b) which is a bit iffy.